src/server/scripts/Kalimdor/BlackfathomDeeps/boss_kelris.cpp

```diff
@@ -47,10 +48,23 @@ class boss_kelris : public CreatureScript
     {
         boss_kelrisAI(Creature* creature) : BossAI(creature, DATA_KELRIS) { }
 
+        void Reset() override
+        {
+            _Reset();
+            DoCastSelf(SPELL_BLACKFATHOM_CHANNELING);
+        }
+
+        void JustReachedHome() override
```

Comment:
Remove this as the reset is fixed

Reply:
Took me a little while to realize that this means "including the next 5 lines".

Reply:
Use suggestion function in github